EFCore.VisualBasic/Design/AnnotationCodeGeneratorProvider/RelationalVisualBasicRuntimeAnnotationCodeGenerator.vb

```diff
@@ -372,12 +393,64 @@ Namespace Design.AnnotationCodeGeneratorProvider
 
                     GenerateSimpleAnnotation(RelationalAnnotationNames.Triggers, triggersVariable, parameters)
                 End If
-
             End If
 
             MyBase.Generate(entityType, parameters)
         End Sub
 
+        Private Sub Create(fragment As IEntityTypeMappingFragment,
+                           fragmentsVariable As String,
+                           parameters As VisualBasicRuntimeAnnotationCodeGeneratorParameters)
+
+            Dim storeObject = fragment.StoreObject
+            Dim code = VBCode
+            Dim overrideVariable = code.Identifier(storeObject.Name & "Fragment", parameters.ScopeVariables, capitalize:=False)
+            Dim mainBuilder = parameters.MainBuilder
+
+            mainBuilder.
+                Append("Dim ").
+                Append(overrideVariable).
+                AppendLine(" As New RuntimeEntityTypeMappingFragment(").
+                IncrementIndent().
+                Append(parameters.TargetName).
+                AppendLine(",")
+
+            AppendLiteral(storeObject, mainBuilder, code)
+
+            mainBuilder.
+                AppendLine(",").
+                Append(code.Literal(fragment.IsTableExcludedFromMigrations)).
+                AppendLine(")").
+                DecrementIndent()
+
+            CreateAnnotations(fragment,
+                              AddressOf Generate,
+                              parameters.Cloner.
+                                         WithTargetName(overrideVariable).
+                                         Clone)
+
+            ' Using reflection because VB currently doesn't seem to be able to call a method that has
+            ' an 'in' parameter if it's virtual.
+            mainBuilder.
+                Append(fragmentsVariable).
+                Append($".GetType().GetMethod(""Add"").Invoke({fragmentsVariable}, {{")
```

Comment:
:disappointed:
I don't know how to call StoreObjects Dictionary.Add() method without using reflection.
VB doesn't seem to be able to call a C# method that has an 'in' parameter if it's virtual.